### ISSUE TITLE
Improve UX for `code_sample` shortcode

### ISSUE THREAD

**Issue by apoorvapendse** [kubernetes/website#48950]
**This is a Feature Request**

**What would you like to be added**

https://github.com/user-attachments/assets/b6fb225f-6061-4235-aab6-d925e9ac5c5b

- The modal's backdrop is frustrating since it blocks the rest of the content unnecessarily.
- Instead of showing a modal on copying text which has to be closed for further usage of the website, a simple animation could be used to denote that the content has been successfully copied to the clipboard.
- Also the placement of the link of the yaml file kept near to copy icon is confusing/inconvenient, maybe the copy icon can be placed at the bottom right corner to prevent misclicks.

**Why is this needed**

Better user experience, current UX is pretty annoying if you want to copy the same content over at multiple places.

**Comments**

**Comment by sftim** [comment]
> can I work on it

Absolutely. Because Kubernetes is open source, you don't even need permission to start work.
